src/runtime/hexagon/rpc/android_bash.sh.template

```diff
@@ -20,7 +20,7 @@ export LD_LIBRARY_PATH=.
 
 # Enable FARF-based logging for Hexagon code invoked by 'tvm_rpc_android_server'.
 export ADSP_LIBRARY_PATH=`pwd`
-echo 0x1f > tvm_rpc_android.farf
+echo 0x1e > tvm_rpc_android.farf
```

Comment:
What is the purpose of this line and why is it changed?

Reply:
Can you add a comment?

Reply:
Done